client/types/automationType.ts

```diff
@@ -1,6 +1,6 @@
 export interface AutomationPreview {
     id: string;
-    name: string;
+    alias: string;
```

Comment:
Should be alias right?

Reply:
No, the swagger specifies that this field is called name. Please follow the documentation we have.